src/r-bridge/lang-4.x/ast/parser/csv/format.ts

```diff
@@ -32,5 +31,19 @@ export function csvToRecord(csv: string[][]): ParsedCsv {
 		entry.token = removeTokenMapQuotationMarks(entry.token)
 		ret[entry.id] = entry
```

Comment:
Maybe a map? (https://rehmat-sayany.medium.com/using-map-over-objects-in-javascript-a-performance-benchmark-ff2f351851ed)